src/index.css

```
@@ -1,5 +1,8 @@
 @import "tailwindcss";
 
+/* Dark mode: triggered by .dark class on <html> */
+@custom-variant dark (&:where(.dark, .dark *));
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)